apiary-receiver/apiary-receiver-sqs/src/test/java/com/expedia/apiary/extensions/receiver/sqs/DefaultSqsMessageDeserializerTest.java

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (C) 2018-2019 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expedia.apiary.extensions.receiver.sqs;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import static com.fasterxml.jackson.databind.DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import com.expedia.apiary.extensions.receiver.common.error.ApiaryReceiverException;
+import com.expedia.apiary.extensions.receiver.common.messaging.MetaStoreEventDeserializer;
+import com.expedia.apiary.extensions.receiver.sqs.messaging.DefaultSqsMessageDeserializer;
+
+@RunWith(MockitoJUnitRunner.class)
+public class DefaultSqsMessageDeserializerTest {
+  private @Mock MetaStoreEventDeserializer delegateSerDe;
+  private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper()
+      .configure(FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Comment:
do we not set this on the default Mapper that we use? Is that correct? @abhimanyugupta07 ?

Reply:
We should have the property set on the default mapper. I have added a comment on `SqsMessageReader`.

In Shunting yard, the mapper was being passed via common beans.